.dev/sass/components/_menus.scss

```diff
@@ -315,6 +315,60 @@ body.no-max-width .main-navigation {
 	}
 }
 
+.navigation.pagination {
+	clear: both;
+	margin: 0 0 1.5em;
+
+	.nav-links {
+		text-align: center;
+
+		.page-numbers {
+			display: inline-block;
+			border: none;
+			border-radius: 3px;
+			background-color: $color__background-body;
+			line-height: 1;
+			margin: 0 0.25em;
+			padding: 0.5em 0.75em;
+			white-space: nowrap;
+
+			&.dots {
+				background: none;
+			}
+
+			&.current {
+				color: $color__background-body;
```

Comment:
@EvanHerman This one is trickier because the Customizer CSS targets from Primer are on `content_background_color` and Activation would need it to be on `background_color`.

What do you think about us changing this entirely so we don't have to make an exception for every child theme? Because most of them don't have two backgrounds like Primer does.

Instead, we could maybe use `button_color` and `button_text_color` on the `.current` selector and `link_color` for everything else.